Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -776,46 +776,38 @@ extension BrowserViewController: WKNavigationDelegate {
 
     func webView(webView: WKWebView, decidePolicyForNavigationAction navigationAction: WKNavigationAction, decisionHandler: (WKNavigationActionPolicy) -> Void) {
         let url = navigationAction.request.URL
-        
-        var openExternally = false
-        
+
         if let scheme = url.scheme {
             switch scheme {
             case "about", "http", "https":
-                openExternally = false
-            case "mailto", "tel", "facetime", "sms":
-                openExternally = true
-            default:
-                // Filter out everything we can't open.
-                decisionHandler(WKNavigationActionPolicy.Cancel)
+                decisionHandler(WKNavigationActionPolicy.Allow)
                 return
+            default:
+                break;
             }
         }
-        
-        if openExternally {
-            if UIApplication.sharedApplication().canOpenURL(url) {
-                // Ask the user if it's okay to open the url with UIApplication.
-                let alert = UIAlertController(
-                    title: String(format: NSLocalizedString("Opening %@", comment:"Opening an external URL"), url),
-                    message: NSLocalizedString("This will open in another application", comment: "Opening an external app"),
-                    preferredStyle: UIAlertControllerStyle.Alert
-                )
 
-                alert.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment:"Alert Cancel Button"), style: UIAlertActionStyle.Cancel, handler: { (action: UIAlertAction!) in
-                    // NOP
-                }))
-                
-                alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment:"Alert OK Button"), style: UIAlertActionStyle.Default, handler: { (action: UIAlertAction!) in
-                    UIApplication.sharedApplication().openURL(url)
-                    return
-                }))
+        if UIApplication.sharedApplication().canOpenURL(url) {
```

Comment:
I'm fine doing those things. Android has a whitelist for things to not prompt for (including SMS). I'm a little nervous about the app store links (i.e. Its a lot easier to get back to Safari from the store than it is to get back to Firefox), and we shouldn't need them (i.e. the App Store redirects to itms:// urls, but apparently they do it incorrectly sometimes).

I'll file some follow ups for this stuff though. This gets us to the point where everything should work (except the evil app store links that aren't using itms://).